test/development/app-dir/source-mapping/app/client/page.tsx

```diff
@@ -1,14 +1,22 @@
 'use client'
 
 import { Form } from '../form'
-import { bar } from '../actions'
+import defaulAction1, { foo, bar, baz, qux } from '../actions1'
```

Comment:
typo nit: `defaultAction` (for all of these)

Reply:
how? those would be duplicate identifiers.

Reply:
`defaul` -> `default`